packages/lodestar-cli/src/commands/dev.ts

```diff
@@ -10,29 +10,32 @@ import PeerId from "peer-id";
 import yaml from "js-yaml";
 import {config as mainnetConfig} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
 import rimraf from "rimraf";
-import {ILogger, WinstonLogger} from "../../logger";
-import {BeaconNode} from "../../node";
-import {BeaconNodeOptions, IBeaconNodeOptions} from "../../node/options";
+import {ILogger, WinstonLogger} from "@chainsafe/eth2.0-utils/lib/logger";
+import {BeaconNode} from "@chainsafe/lodestar/lib/node";
+import {IBeaconNodeOptions} from "@chainsafe/lodestar/lib/node/options";
 import {generateCommanderOptions, optionsToConfig,} from "../util";
-import {getTomlConfig, rmDir} from "../../util/file";
+import {rmDir} from "@chainsafe/lodestar/lib/util/file";
 import {config as minimalConfig} from "@chainsafe/eth2.0-config/lib/presets/minimal";
-import {InteropEth1Notifier} from "../../eth1/impl/interop";
-import {quickStartOptionToState} from "../../interop/cli";
+import {InteropEth1Notifier} from "@chainsafe/lodestar/lib/eth1/impl/interop";
+import {quickStartOptionToState} from "../lodestar/interop/cli";
```

Comment:
We shouldn't have any relative path references across packages.
They should all be referenced through the package name.

Reply:
`lodestar` is a folder name under `src` inside the cli package. The idea is to have package name level in folder tree so that we can move files from other package to cli if needed.

Reply:
ohhh, gotcha